contracts/seed/Seed.sol

```diff
@@ -34,8 +34,8 @@ contract Seed {
     address public admin;
     uint256 public softCap;
     uint256 public hardCap;
-    uint256 public seedAmountRequired; // Amount of seed required for distribution
-    uint256 public feeAmountRequired; // Amount of seed required for fee
+    uint256 public seedAmountRequired; // Amount of seed required for distribution (buyable + tip)
```

Comment:
buyable = totalBuyableSeed?
If yes, can this be renamed to the latter?

Reply:
@Zitzak Decided against renaming?

No need to discuss now, but in the long run
I would prefer if we use exact variables, eg. "buyableSeed" instead of "buyable" or "buyable seed".